src/docfx/build/Build.cs

```diff
@@ -196,9 +196,9 @@ void ValidateBookmarks()
                 {
                     SourcePath = file.FilePath,
                     SiteUrl = file.SiteUrl,
-                    Monikers = monikers,
                     OutputPath = GetOutputPath(file, monikers),
                 };
+                manifest.Monikers.AddRange(monikers);
```

Comment:
> manifest.Monikers.AddRange(monikers); [](start = 15, length = 38)

why doing that?

Reply:
Because I want to initialize a list to `Monikers`, then we do not need to check null anywhere